torch/csrc/api/include/torch/nn/functional.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <torch/nn/functional/distance.h>
-#include <torch/nn/functional/loss.h>
-#include <torch/nn/functional/pooling.h>
 #include <torch/nn/functional/embedding.h>
+#include <torch/nn/functional/loss.h>
+#include <torch/nn/functional/normalization.h>
+#include <torch/nn/functional/pooling.h>
```

Comment:
nit: missing new line here

Reply:
In general all C++ files should be padded with a newline? I think I got a bit confused on whether it applies everywhere or whether it follows a pattern